sync/fromDisk/questions.js

```diff
@@ -24,6 +26,22 @@ module.exports.sync = function(courseInfo, questionDB, jobLogger, callback) {
         const questionsParam = Object.keys(questionDB).map(qid => {
             const q = questionDB[qid];
 
+            var thumbnailPresence = false;
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.jpg'))) {
+                thumbnailPresence = true;
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.png'))) {
+                thumbnailPresence = true;
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.jpeg'))) {
+                thumbnailPresence = true;
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.svg'))) {
+                thumbnailPresence = true;
+            }
+            if (fs.existsSync(path.join(courseInfo.questionsDir, qid, 'thumbnail.gif'))) {
+                thumbnailPresence = true;
+            }
```

Comment:
I see that these lines are essentially duplicated here and in `questionThumbnail.js`, would we be able to find the path once and then save that for whenever we request the thumbnail?  Perhaps instead of a thumbnail boolean, we could store the path to the thumbnail in the db and then use that in the express router?